### PR TITLE
add ceph StorageClass example

### PR DESCRIPTION
**Pull request by jjmengze**
**What this PR does / why we need it**:

add ceph StorageClass example 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: cluster/addons/storage-class/caph/default.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: ceph
```
Comment (dixudx) [inline cluster/addons/storage-class/caph/default.yaml]: append below contents ?

```
  annotations:
    storageclass.beta.kubernetes.io/is-default-class: "true"
  labels:
    addonmanager.kubernetes.io/mode: Reconcile
```

File: cluster/addons/storage-class/caph/default.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+provisioner: kubernetes.io/rbd
+# parameters:
```
Comment (dixudx) [inline cluster/addons/storage-class/caph/default.yaml]: Do we really need these comments?

File: cluster/addons/storage-class/caph/default.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
```
Comment (krmayankk) [inline cluster/addons/storage-class/caph/default.yaml]: why do we need a StorageClass in an addons folder ?

Comment (MengZn) [reply cluster/addons/storage-class/caph/default.yaml]: in this ref https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/storage-class
When we need to use external storage, one more ceph option is available.

File: cluster/addons/storage-class/caph/default.yaml (hunk 2 above)
Comment (MengZn) [reply cluster/addons/storage-class/caph/default.yaml]: @dixux What advice can be given to me to make it better.

**Comment by MengZn** [comment]
@zhangxiaoyu-zidif is this a right solution ?

**Pull request closed without merging**